MAUI/ImageEditor/save.md

```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+## Reset
+
+The [`Reset`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_Reset) method, resets an image to its original state by discarding the changes performed to it. You can start fresh and explore different creative possibilities.the complete set of changes made in image and also resets the image to original loaded image.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Text="Reset"
+                Clicked="OnResetClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnResetClicked(object sender, EventArgs e)
+    {
+        this.imageEditor.Reset();
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Reset events
+
+The SfImageEditor has events when performing reset operation namely [`BeginReset`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_BeginReset).
```

Comment:
The SfImageEditor has event when performing reset operation namely [`BeginReset`]

Reply:
Changed a content